src/main/java/io/spaship/operator/service/k8s/Operator.java

```diff
@@ -37,414 +36,413 @@
-    var ns = (environment.getNameSpace()).replace(tenantName.concat("--"), "");
-    LOG.debug("Creating namespace with name {}", ns);
-    var appCode = ConfigProvider.getConfig().getValue("mpp.app.code", String.class);
-
-    var devOpsNamingConvention = ConfigProvider.getConfig()
-        .getValue("application.devops.naming.convention", String.class);
-
-    return Map.of("APP_CODE", appCode,
-        "TENANT_NAME", tenantName,
-        "NS_NAME", ns,
-        "DEVOPS_NAMING_CONVENTION", devOpsNamingConvention,
-        "DE_NAMESPACE", deDebugNs);
-  }
-
-  private void createNewTenantNamespace(Environment environment, Map<String, String> templateParameters) {
-    var k8sNSList = ocClient
-        .templates()
-        .load(Operations.class.getResourceAsStream("/openshift/mpp-namespace-template.yaml"))
-        .processLocally(templateParameters);
-    ocClient.resourceList(k8sNSList).createOrReplace();
-    LOG.debug("new namespace {} created successfully ", environment.getNameSpace());
-  }
-
-  @SneakyThrows
-  private void prepareNewTenantNameSpace(Environment environment, Map<String, String> templateParameters,
-      EventStructure.EventStructureBuilder eb) {
-    var nsSupportResourcesList = ocClient
-        .templates()
-        .inNamespace(environment.getNameSpace())
-        .load(Operations.class.getResourceAsStream("/openshift/mpp-prepare-namespace.yaml"))
-        .processLocally(templateParameters);
-
-    Uni.createFrom().item(nsSupportResourcesList)
-        .map(item -> {
-          LOG.debug("executing mpp-prepare-namespace.yaml for namespace {}", environment.getNameSpace());
-          return ocClient.resourceList(item).inNamespace(environment.getNameSpace()).createOrReplace();
-        })
-        .onFailure()
-        .retry()
-        .withBackOff(Duration.ofSeconds(1), Duration.ofSeconds(2))
-        .atMost(10)
-        .onFailure()
-        .recoverWithItem(throwable -> {
-          throwable.printStackTrace();
-          return null;
-        }).subscribeAsCompletionStage()
-        .thenAccept(reactOnOperationOutcome(environment, eb)).get();
-    updateTenantEgressForIAD2Cluster(environment.getNameSpace());
-  }
-
-  private void updateTenantEgressForIAD2Cluster(String namespace) {
-    var domainName = ConfigProvider.getConfig().getValue("operator.domain.name", String.class);
-    if(!domainName.contains("iad2")){
-      LOG.info("This namespace is not in iad2 cluster hence skipping the TenantEgress update process");
-      return;
+
+    @SneakyThrows
+    private void prepareNewTenantNameSpace(Environment environment, Map<String, String> templateParameters,
+                                           EventStructure.EventStructureBuilder eb) {
+        var nsSupportResourcesList = ocClient
+                .templates()
+                .inNamespace(environment.getNameSpace())
+                .load(Operations.class.getResourceAsStream("/openshift/mpp-prepare-namespace.yaml"))
```

Comment:
Read it from the variable 

could you please add this path to a variable ? (also if possible you can provide an option to read this path from the env variable if you think it's feasible)

`private final string OPENSHIFT_MPP_PREPARE_NAMESPACE_PATH = "/openshift/mpp-namespace-template.yaml";`